src/main/java/bl4ckscor3/mod/scserverutils/ChatMuteHandler.java

```diff
@@ -0,0 +1,24 @@
+package bl4ckscor3.mod.scserverutils;
+
+import net.minecraft.network.chat.Component;
+import net.minecraft.server.level.ServerPlayer;
+import net.neoforged.bus.api.IEventBus;
+import net.neoforged.neoforge.common.NeoForge;
+import net.neoforged.neoforge.event.ServerChatEvent;
+
+public class ChatMuteHandler {
+
+    public static void register(IEventBus modEventBus) {
+
+    }
+
+    public static void onPlayerChat(ServerChatEvent event) {
```

Comment:
```suggestion
    public static void onServerChat(ServerChatEvent event) {
```

Also, since this class only really has one handler, it can probably be removed and the handler just be put at the bottom of `SCServerUtils`. That also removes the need of registering this handler method. As you can see, there are more event handlers there that don't really have anything else that needs to be done.